pkg/aws_config_client/completer.go

```diff
@@ -143,31 +226,47 @@ func (c *completer) writeAWSProfile(out *ini.File, region string, profile *AWSNa
 	return nil
 }
 
+func (c *completer) writeAWSProfiles(out *ini.File, region string, profiles []*AWSNamedProfile) error {
```

Comment:
feels like this function is [duplicate](https://github.com/chanzuckerberg/aws-oidc/blob/master/pkg/aws_config_client/completer.go#L124)